addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/bbqtemperature.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rfxcom" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="bbqtemperature">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+			<bridge-type-ref id="tcpbridge" />
+			<bridge-type-ref id="RFXtrx433" />
+			<bridge-type-ref id="RFXrec433" />
+		</supported-bridge-type-refs>
+
+		<label>RFXCOM BBQ Temperature Sensor</label>
+		<description>A BBQ Temperature device.</description>
+
+		<channels>
+			<channel id="foodTemperature" typeId="temperature" />
+			<channel id="bbqTemperature" typeId="temperature" />
```

Comment:
I tried to document the answer here: https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channels

So assuming that hopefully food temperature is semantically different to bbq temperature, it should be indeed two different channel types. (and yes, sorry, the wind thing is probably a bad example then, because this mistake slipped through the review at that time 🙁).